catalogue_pipeline/Makefile

```diff
@@ -5,11 +5,16 @@ STACK_ROOT 	= catalogue_pipeline
 
 SBT_APPS 	= id_minter ingestor reindex_worker transformer
 ECS_TASKS 	=
-LAMBDAS 	= schedule_reindexer reindexer_v2/complete_reindex reindexer_v2/reindex_job_creator
+LAMBDAS 	= reindexer_v2/complete_reindex \
+              reindexer_v2/reindex_job_creator \
+              reindexer_v2/reindex_shard_generator
 
 TF_NAME 	= catalogue_pipeline
 TF_PATH 	= $(ROOT)/catalogue_pipeline/terraform
 
 TF_IS_PUBLIC_FACING = false
 
+reindexer_lambdas-test: reindexer_v2/reindex_job_creator-test \
```

Comment:
I think we can delete this Make task; it was meant to be used in Travis but in practice we're just doing separate tests for each Lambda.